src/parachain/issue.ts

```diff
@@ -213,8 +342,16 @@ export class DefaultIssueAPI implements IssueAPI {
     }
 
     async getRequestById(issueId: H256): Promise<IssueRequestExt> {
+        return (await this.getRequestsByIds([issueId]))[0];
```

Comment:
can you add in the head part to only take this from the finalized one?

Reply:
Not sure what you mean - do you mean in the doc comment? What do you mean by "only take this from the finalized one"?
(In case it's not clear, it's the same functionality and logic as it always has been, it just now works by wrapping an array-capable version (to avoid duplicating the same logic in both).)

Reply:
ah, nevermind. I missed that it calls the function below.